### DISCUSSION TITLE
Cookies:  .DOTNETNUKE and dnn_IsMobile

### DISCUSSION THREAD

**Discussion by thabaum** [dnnsoftware/Dnn.Platform#4043] [General] [answered]
I am curious how to edit the names of these cookies:

![image](https://user-images.githubusercontent.com/13577556/92166040-d03dd780-edec-11ea-9e91-a858437c73ab.png)

I did it in the web config however they still appear.

Is there a way to adjust these or are they hard coded into DNN?

**Comment by bdukes** [comment, marked as answer]
You should be able to rename `.DOTNETNUKE` via the `web.config` (renaming the cookie wouldn't delete the old cookie), but `dnn_IsMobile` appears to be hardcoded.

**Comment by thabaum** [comment]
Thanks @bdukes the rename after I deleted the cookie shows only the new cookie name.  

The `dnn_IsMobile` cookie, do you think it be ok to rename this to something more generic in the app like `app_IsMobile` and create a PR?

**Reply by thabaum** [reply]
`private const string MobileViewSiteCookieName = "dnn_IsMobile";
 private const string DisableMobileViewCookieName = "dnn_NoMobile";`

Line 33 and 34

I will put together a PR For this it should not be a breaking change see if community will support.

**Reply by bdukes** [reply]
I don't know that renaming on its own is going to be enough of a change to get a PR accepted.  If the cookie name was optionally pulled from some configuration source (`web.config`, host setting, portal setting, etc.), that would probably be something that could get accepted.

**Reply by thabaum** [reply]
I went ahead and put one together seemed simple enough just to change the name #4045 

If no one likes the idea I can go ahead and research more into a bigger fix that allows changing it.   I do not see this PR posted as a breaking change and gets the job done to my satisfaction.  

I do like having control over naming it in the web config so maybe a future enhancement could be to make that happen.

**Reply by thabaum** [reply]
One other issue is all portals share the same cookie name.  So if you change `.DOTNETNUKE` to `.MYWEBAPP` all your sites will display that same cookie name.  

Maybe this should get changed to something more generic as a default as well?
